Python/Product/VSInterpreters/Interpreter/CondaEnvironmentFactoryProvider.cs

```diff
@@ -295,9 +300,13 @@ class CondaEnvironmentFactoryProvider : IPythonInterpreterFactoryProvider, IDisp
 
         // Gets the python package version installed in a specified conda environment
         internal async static Task<Version> ExecuteCondaListAsync(string condaPath, string envName) {
+            
+            // Make sure we're on thread pool thread before synchronously blocking on Conda's output
+            await TaskScheduler.Default;
+
             Version version = null;
+            var activationVars = await CondaUtils.GetActivationEnvironmentVariablesForRootAsync(condaPath);
 
-            var activationVars = await CondaUtils.GetActivationEnvironmentVariablesForRootAsync(condaPath).ConfigureAwait(false);
```

Comment:
ConfigureAwait is unneeded, this will resume on thread pool due to await TaskScheduler.Default above it.